ecleankernel/file.py

```diff
@@ -91,6 +96,42 @@ def __init__(self,
         super().__init__(path, KernelFileType.KERNEL)
         self.internal_version = self.read_internal_version()
 
+    def __decompress_raw(self) -> bytes:
```

Comment:
Double underscore is for special Python methods, just name it normally here.

Reply:
I thought __ would mark "private" functions - fixed